File: application/src/main/java/com/study/platform/domain/comment/usecase/UpdateCommentUseCase.java
package com.study.platform.domain.comment.usecase;

import com.study.platform.domain.comment.dto.request.CommentUpdateRequest;
import com.study.platform.domain.comment.dto.response.CommentResponse;

import java.util.UUID;

public interface UpdateCommentUseCase {
    CommentResponse execute(UUID userId, UUID commentId, CommentUpdateRequest request);
}

File: application/src/main/kotlin/com/study/platform/domain/comment/usecase/UpdateCommentUseCase.kt
package com.study.platform.domain.comment.usecase

import com.study.platform.domain.comment.dto.request.CommentUpdateRequest
import com.study.platform.domain.comment.dto.response.CommentResponse
import java.util.UUID

interface UpdateCommentUseCase {
    fun execute(userId: UUID, commentId: UUID, request: CommentUpdateRequest): CommentResponse
}
